bin/all_sky_search/pycbc_upload_single_event_to_gracedb

```diff
@@ -0,0 +1,195 @@
+from ligo.gracedb.rest import GraceDb
+import numpy as np
+
+import pycbc
+import lal
+from pycbc.io.live import gracedb_tag_with_version
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--xml-file-for-upload', required=True, type=str,
+                    help='LIGOLW XML file containing the event.')
+parser.add_argument('--log-message', type=str, metavar='MESSAGE',
+                    help='Add a log entry to each upload with the given message')
+parser.add_argument('--testing', action="store_true", default=False,
+                    help="Upload event to the TEST group of gracedb.")
+parser.add_argument('--production-server', action="store_true", default=False,
+                    help="Upload event to production graceDB. If not given "
+                         "events will be uploaded to playground server.")
+parser.add_argument('--force-overwrite', action='store_true', default=False,
+                    help="GraceDb instance will be checked for if an event "
+                         "with the same event time and FAR already exist. "
+                         "If so, event will not be uploaded")
+parser.add_argument('--query-string', default='pycbc',
+                    help="If not using --force-overwrite, add a string to "
+                         "gracedb query to further filter events. "
+                         "Default='pycbc'")
+parser.add_argument('--search-id-string', default='AllSky',
+                    help="Using T050017-v1 naming convention for "
+                         "output XML filename. This is a string for "
+                         "search identifier in filename, e.g. 'AllSky' "
+                         "would give H1L1V1-PYCBC_AllSky-1234567890-1.xml. "
+                         "See https://dcc.ligo.org/LIGO-T050017/public. "
+                         "Default: 'AllSky'")
+parser.add_argument('--snr-timeseries-plot',
+                    help="SNR timeseries plot to be uploaded with the event.")
+parser.add_argument('--asd-plot',
+                    help="ASD plot to be uploaded with the event.")
+parser.add_argument('--skymap-plot',
+                    help="Skymap plot to be uploaded with the event.")
+parser.add_argument('--skymap-fits-file',
+                    help="Skymap .fits file to be uploaded with the event.")
+parser.add_argument('--source-probabilities',
+                    help="Source probabilities file to be uploaded with the "
+                         "event.")
+parser.add_argument('--source-probabilities-plot',
+                    help="Source probabilities plot to be uploaded with the "
+                          "event.")
+parser.add_argument('--labels', nargs='+',
+                    help="Labels to add to the event in GraceDB")
+
+args = parser.parse_args()
+
+pycbc.init_logging(logging.INFO)
+# Make the scitokens logger a little quieter
+# (it is called through GraceDB)
+logging.getLogger('scitokens').setLevel(logging.root.level + 10)
```

Comment:
Does the scitoken logger always exist, and if not, does this cause the code to crash?

Reply:
I've tested and this works with a generic string, i.e.:
```
In [1]: import logging                                                                               

In [2]: logging.getLogger('something')                                                               
Out[2]: <Logger something (WARNING)>                                                                                             

In [3]: logging.getLogger('something').setLevel(40)                                                  

In [4]: logging.getLogger('something')                                                               
Out[4]: <Logger something (ERROR)>
```

So it won't crash